projects/storefrontlib/src/cms-components/myaccount/order/order-history/order-history.component.html

```diff
@@ -1,9 +1,11 @@
 <ng-container *ngIf="orders$ | async as orders">
   <div class="container">
     <!-- HEADER -->
+    <!--
```

Comment:
I just comment it out. `Header` should be displayed when the feature is disabled. Toggle feature will be handled in #4659